app/javascript/src/components/Player.tsx

```diff
@@ -30,6 +30,7 @@ interface PlayerProps {
   story: Object
   theme: string
   title: string
+  isOffline: boolean
```

Comment:
this variable ever used?

Reply:
I believe so -- let me try to remove it and see if it actually is needed

Reply:
It is used -- not entirely why but it errors on compile without it